tests/test_config.py

```diff
@@ -299,9 +299,7 @@ def test_clean_added_user_no_bmo_id_with_refresh(
 
     assert file_config.config["people"] == config["people"]
     file_config.config["modules"][0]["owners"].append({"nick": "TLMC"})
-    file_config.config["modules"][0]["submodules"][0]["owners"].append(
-        {"nick": "TLMC"}
-    )
+    file_config.config["modules"][0]["submodules"][0]["owners"].append({"nick": "TLMC"})
```

Comment:
Hmm, interesting that this change popped up here. There might be a divergence between the tests and `make format`

Reply:
I'll file a separate bug for this, the tests directory is missing from the test lint paths, but is present in the format command.